packages/docusaurus/src/client/serverEntry.js

```diff
@@ -87,4 +88,16 @@ export default async function render(locals) {
       rmWhitespace: true,
     },
   );
+
+  // minify html with https://github.com/DanielRuf/html-minifier-terser
+  return minify(renderedHtml, {
+    collapseWhitespace: true,
```

Comment:
How about adding these two options?

- removeAttributeQuotes
- removeEmptyAttributes

Reply:
I think removeAttributeQuotes can be quite unsafe according to http://jkorpela.fi/qattr.html and HTML specification recommends to always use quotes.

Reply:
As I understand correctly, not all quotes are removed, but only in places where this is possible? But OK, let's do without this option.